src/organisms/tables/_ComparisonTable/index.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
```

Comment:
Since `_ComparisonTable` isn't being used anywhere I think, we could probably just delete it's files.